contracts/cheddar_5x5_tic_tac_toe/src/lib.rs

```diff
@@ -108,6 +120,7 @@ impl Contract {
     pub fn make_available(
         &mut self,
         game_config: Option<GameConfigNear>,
+        available_for: u64,
```

Comment:
we need to check that available_for is bigger than 1min and smaller then whatever makes sense (let's say 1h)

Reply:
DONE